components/list/list-item-generic-layout.js

```diff
@@ -248,6 +258,13 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 		`;
 	}
 
+	updated(changedProperties) {
+		super.updated(changedProperties);
+		if (changedProperties.has('indentation')) {
+			this.style.setProperty('--d2l-list-item-generic-layout-nested-indentation', `${this.indentation}px`);
```

Comment:
Do we want to do this in `willUpdate` instead? I'm not sure we'll reduce any layout though, since this is effectively updating the DOM outside of `render` either way, but would happen earlier.